webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -90,6 +90,17 @@ def timestamp_to_date(
 
 
 @library.filter
+def time_ago(dt, format='%a, %b %d %H:%M %Z'):
+    return jinja2.Markup(
+        '<time datetime="{}" class="ago">{}</time>'
+        .format(
+            dt.isoformat(),
+            dt.strftime(format)
+        )
+    )
+
+
```

Comment:
If we only ever generate `<time class="ago">` from django, how come `timeutils.js` is iterating over `$('time.in')`. Where do they get created?